src/server/pkg/storage/fileset/fileset_test.go

```diff
@@ -123,7 +123,7 @@ func TestWriteThenRead(t *testing.T) {
 		for _, fileName := range fileNames {
 			data := chunk.RandSeq(rand.Intn(max))
 			files = append(files, &testFile{
-				name: fileName,
+				name: "/" + fileName,
```

Comment:
We shouldn't need this anymore right? Since we are canonicalizing and applying cleaning to requests.

Reply:
It validates based on what's in files.